config/deploy.rb

```diff
@@ -17,7 +17,7 @@ def main_deploy_server
 set :rails_env, fetch(:stage)
 set :rvm1_map_bins, -> { fetch(:rvm_map_bins).to_a.concat(%w[rake gem bundle ruby]).uniq }
 
-set :application, "consul"
+set :application, deploysecret(:app_name)
```

Comment:
For easier compatibility with existing installations, how about:

```suggestion
set :application, deploysecret(:app_name) || "consul"
```

This way we don't have to ask people to change their `deploy-secrets.yml` file when upgrading.